spec/lib/video_info_spec.rb

```diff
@@ -70,6 +70,13 @@
     end
   end
 
+  describe ".provider_api_keys" do
+    it "raises an error if key isn't a symbol" do
+      expect { VideoInfo.provider_api_keys = { "Youtube" => "key" } }.to raise_error(ArgumentError)
+      expect { VideoInfo.provider_api_keys = { youtube: "key" } }.to_not raise_error
```

Comment:
Line is too long. [84/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.